fix checkbox display

Jotakuun opened the pull request:
**What kind of change does this PR introduce?** (check one with "x")
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
#943 

**What is the new behavior?**
I just removed the display none from the material.scss.


**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...


**Other information**:

Jotakuun commented:
I don't know why checkbox and radio inputs wasn't displaying because if there is a reason to hide it or a new behaviour just tell me and I'll fix it and update the pull request.

wizarrc commented:
@Jotakuun Line 1 - 8 should never have been added. It looks like it was accidently added from the grouping rows PR. That stuff should be moved to the demo css file.

wizarrc commented:
I suggest moving line 1-8 to https://github.com/swimlane/ngx-datatable/blob/master/assets/app.css next to `.expectedpayments` section, and uncomment line 1 and 2.

Jotakuun commented:
Ok, thank you @wizarrc 
I removed the comments I left in material.scss and I moved those lines to the app.css 

By the way, those classes and ids selectors doesn't match to the row-grouping checkboxes anymore. So I named it like they are now. But anyway, row-grouping demo is not working because all labels are recieving same name and id from rowIndex. I'm going to investigate what's happening and let's see if I can fix it.

wizarrc commented on src/components/body/body.component.ts:
```diff
@@ -682,7 +682,8 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
    */
   getRowIndex(row: any): number {
-    return this.rowIndexes.get(row) || 0;
+    // return this.rowIndexes.get(row) || 0;
```
@Jotakuun The `indexOf` array function is much slower than the Map `get` function. Are you having issues with it returning the wrong value?

Jotakuun replied:
Ok, so it looks like the getRowIndex() was changed in last release and this piece of code wasn't working for rowGroups because rowIndexes is not returning any index for rowGroups. I couldn't figure it out why. Any suggestion?

wizarrc replied:
hmm, not sure. Didn't really get a chance to look over the row groups PR that much. I can tell you that `rowIndexes` should have a list of indexes per row, same as the `rows` array being searched manually, but much slower. If `rowIndexes` isn't producing correct results, then it's not being updated correctly.

amcdnl replied:
Grouped row indexes are in a different collection.

Jotakuun replied:
And where is it? The ```rowIndexes``` are not updating in the ```updateRow()``` method when there's groupedRows.

wizarrc replied:
Can we treat this is a separate issue? I find value in moving the css out of the `material.scss` as it's causing issues.

amcdnl replied:
i removed it and released thisx morning.

Jotakuun replied:
I agree, this is completely another issue. I update the PR again with just the ```material.scss``` and we treat this in another pull request.

wizarrc commented:
related issue #939

iamchiranjeevin commented:
im using latest version i still cant see the checkboxes
